server/src/services/metadata.service.ts

```diff
@@ -589,9 +589,18 @@ export class MetadataService extends BaseService {
     let dateTimeOriginal = dateTime?.toDate();
     let localDateTime = dateTime?.toDateTime().setZone('UTC', { keepLocalTime: true }).toJSDate();
     if (!localDateTime || !dateTimeOriginal) {
-      this.logger.warn(`Asset ${asset.id} has no valid date, falling back to asset.fileCreatedAt`);
-      dateTimeOriginal = asset.fileCreatedAt;
-      localDateTime = asset.fileCreatedAt;
+      // When a file is copied (but not moved) before being uploaded to immich, the target file creation
+      // date is set at the current timestamp, while the modification date remains untouched, so if the
+      // user copied the asset while he did not modified the file (like cropping, rotating and more), then
+      // we use the modification timestamp as it's still the original date. If the user modified the asset,
+      // then there is no other solution except a further manual fix.
+      this.logger.warn(
```

Comment:
@alextran1502 is `warn` the good level ? I would more see it as `debug` as it's because of the user asset not exif content, but not a server error or abnormal behavior

Reply:
@danieldietzler also if you have an idea

Reply:
Yeah debug is fine

Reply:
```suggestion
      this.logger.debug(
```